lacework/resource_lacework_integration_ecr.go

```diff
@@ -587,7 +587,10 @@ func readEcrIam(d *schema.ResourceData, meta interface{}) error {
 		d.Set("limit_num_imgs", response.Data.Data.LimitNumImg)
 		d.Set("limit_by_tags", response.Data.Data.LimitByTag)
 		d.Set("limit_by_repositories", response.Data.Data.LimitByRep)
-		d.Set("limit_by_label", castArrayOfStringKeyMapOfStringsToLimitByLabelSet(response.Data.Data.LimitByLabel))
+
+		if limitByLabelsLength(response.Data.Data.LimitByLabel) != 0 {
```

Comment:
Wouldn't we need this same check for all the places where `limit_by_label` is used? For example GCR, GAR, GHCR, Docker?